cmds/script.c

```diff
@@ -24,7 +24,7 @@ extern char script[];
 
 static void cmd_scriptInfo(void)
 {
-	lib_printf("shows script, usage: script <preinit/name> [dev] [magic]");
+	lib_printf("shows script, usage: script <[dev] [name] [magic]>");
```

Comment:
```suggestion
	lib_printf("shows script, usage: script [<dev> <name> <magic>]");
```
There may be no arguments or there must be 3.

Reply:
Done